hangups/ui/__main__.py

```diff
@@ -198,11 +198,23 @@ def _on_event(self, conv_event):
 
     def _on_quit(self):
         """Handle the user quitting the application."""
-        future = asyncio.async(self._client.disconnect())
-        future.add_done_callback(lambda future: future.result())
+        asyncio.ensure_future(self._client.disconnect())
```

Comment:
`async.ensure_future` was only added in Python 3.4.4, so this would require raising the minimum requirement. Not sure if it's worthwhile for now.